### PR TITLE
Updates to match template (README)

### PR DESCRIPTION
**Pull request by sinsukehlab**
### Why:

This course is different from the template.

### What's being changed:

Add the **Start course** button
Some minor changes

### Check off the following:

- [x] For workflow changes, I have verified the Actions workflows function as expected.
- [x] For content changes, I have reviewed the [style guide](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### CODE REVIEW

**Comment by sinsukehlab** [comment]
#2 &rarr; 404 :thinking:

File: README.md [hunk 1]
```diff
@@ -1,28 +1,47 @@
...
+- **What you'll learn**: How to create a codespace, push code from a codespace, select a custom image, and customize a codespace.
+- **What you'll build**: A codespace with devcontainer.json files, customizations, and personalizations.
+- **Prerequisites**: If you need to learn about Visual Studio Code, read [Visual Studio Code Docs](https://code.visualstudio.com/docs) first.
+- **How long**: This course is four steps long and can be completed in less than an hour.
 
-<!--endstep0-->
+## How to start this course
 
+1. Right-click **Start course** and open the link in a new tab.
+   <br />[![start-course](https://user-images.githubusercontent.com/1221423/218596841-0645fe1a-4aaf-4f51-9ab3-8aa2d3fdd487.svg)](generate)
```
Comment (heiskr) [inline README.md:37]:
```suggestion
   <br />[![start-course](https://user-images.githubusercontent.com/1221423/218596841-0645fe1a-4aaf-4f51-9ab3-8aa2d3fdd487.svg)](https://github.com/skills/code-with-codespaces)
```

**Review by heiskr — APPROVED** [review]
Looks amazing, thank you 🙏🏼

File: README.md [hunk 2]
```diff
@@ -1,28 +1,47 @@
...
+  Include a 1280×640 image, course name in sentence case, and a concise description in emphasis.
+  In your repository settings: enable template repository, add your 1280×640 social image, auto delete head branches.
+  Next to "About", add description & tags; disable releases, packages, & environments.
+  Add your open source license, GitHub uses the MIT license.
+-->
+
 # Code with GitHub Codespaces and Visual Studio Code
 
-<!--step0-->
+_Create workflows that enable you to use Continuous Integration (CI) for your projects._
```
Comment (sinsukehlab) [inline README.md:12]: corrected in #25

**Pull request merged**